test/queue-test.js

```diff
@@ -572,6 +572,126 @@ describe('Queue', (it) => {
     });
   });
 
+  it.describe('getJobs', (it) => {
+    it('gets waiting jobs', async (t) => {
+      const queue = t.context.makeQueue();
+
+      const insertedJobs = [];
+      for (let i = 0; i < 3; i++) {
+        let job = await queue.createJob({foo: 'bar'}).save();
+        insertedJobs.push(job.toData());
+      }
+
+      const jobs = await queue.getJobs('waiting', {start: 0, end: 2});
+      for (let job of jobs) {
+        t.true(insertedJobs.includes(job.toData()));
+      }
```

Comment:
I'd prefer that this check the `id` instead of the data, if possible - it seems that this won't be a good test because they will all have the same data